diff --git a/configure.ac b/configure.ac
index 822aaba..890e3bd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -767,7 +767,7 @@ AC_CHECK_FUNCS([DINFINITY DQNAN cexp chsize clog clog10 ctermid		\
   strdup system usleep atexit on_exit chown link fcntl ttyname getpwent	\
   getgrent kill getppid getpgrp fork setitimer getitimer strchr strcmp	\
   index bcopy memcpy rindex truncate unsetenv isblank _NSGetEnviron	\
-  strcoll strcoll_l newlocale utimensat sched_getaffinity		\
+  strcoll strtol_l newlocale utimensat sched_getaffinity		\
   sched_setaffinity sendfile])
 
 AM_CONDITIONAL([BUILD_ICE_9_POPEN],
